astropy/io/fits/scripts/fitsdiff.py

```diff
@@ -43,7 +43,7 @@
 Example
 -------
 
-    fitsdiff -k filename,filtnam1 -n 5 -d 1.e-6 test1.fits test2
+    fitsdiff -k filename,filtnam1 -n 5 -r 1.e-6 test1.fits test2
```

Comment:
If they are not yet CI tested, they should be somehow included in there. That can be part of another PR, but it shouldn't be forgotten, having broken command line tools is much much worse than not having them at al.